src/Internal/EventStorePersistentSubscription.php

```diff
@@ -60,7 +60,7 @@ class EventStorePersistentSubscription implements AsyncEventStorePersistentSubsc
     private ?DropData $dropData = null;
     private bool $isDropped = false;
     private int $bufferSize;
-    private ManualResetEventSlim $stopped;
+    private Deferred $stopped;
```

Comment:
So... this is what I can use to check if a subscription is active, right? That should solve my problem in #117.

Reply:
Can't wait for it. :+1: